website/docs/r/app_service_connection.html.markdown

```diff
@@ -112,6 +112,34 @@ An `authentication` block supports the following:
 
 ---
 
+* `scope` - (Optional) Connection scope in source service.
```

Comment:
Is it possible to get the valid values here to assist users in completing this part of the config?

Reply:
I'll double check with the service team.

Reply:
Added some more info provided by the service team.